### DISCUSSION TITLE
https://feat-enable-markdown.proof-arena-web.pages.dev/problemsDetail/1002

### DISCUSSION THREAD

**Discussion by giscus** [PolyhedraZK/proof-arena#6] [General]
# https://feat-enable-markdown.proof-arena-web.pages.dev/problemsDetail/1002

https://feat-enable-markdown.proof-arena-web.pages.dev/problemsDetail/1002

**Comment by GassaPolyhedra** [comment]
Hello discussion in feat testing

**Comment by niconiconi** [comment]
test
